api_types.ts

```diff
@@ -449,6 +450,7 @@ export type SuggestedValueType<T extends UnionType> = T extends ArrayType<Type.d
   : TypeOfMap<T>;
 
 // TODO(patrick): Unhide this
+/** @deprecated use `ParentDefinition` instead */
```

Comment:
Ditto. But SyncTableRelation.inheritPermissions is now deprecated.

Reply:
Feel free to mark ` SyncTableRelation.inheritPermissions` as deprecated though unless you're waiting.

Reply:
Done.